lib/freeclimb/utils/request_verifier.rb

```diff
@@ -0,0 +1,57 @@
+module Freeclimb
+    class RequestVerifier
+        @@DEFAULT_TOLERANCE = 5 * 60 * 1000
+        attr_accessor :info
```

Comment:
Not needed. Specification does not define this attribute is required

Reply:
will remove in latest commit